zebra/zebra_router.c

```diff
@@ -163,6 +163,15 @@ void zebra_router_sweep_route(void)
 static void zebra_router_free_table(struct zebra_router_table *zrt)
 {
 	void *table_info;
+	struct zebra_vrf *zvrf;
+
+	zvrf = zvrf_lookup_from_attribute(zrt->tableid, zrt->ns_id);
+	if (!zvrf)
+		zlog_warn("zrouter: vrf for table %u, ns %u not found. cancel rib_close.",
```

Comment:
Hate to ask this but ... is there a rule about introducing new raw `zlog_warn()`, instead of  allocating new `flog_warn` codes?

Reply:
you are right.
I think it is better to call flog_warn() instead